### ISSUE TITLE
Allow Configurable Unity Version

### ISSUE THREAD

**Issue by Rabadash8820** [webbertakken/unity-test-runner#9]
First off, your Unity Actions are fantastic, thank you so much for building these!

Would it be possible to make the Unity Version for this action configurable, like it is for the [Unity - Builder](https://github.com/marketplace/actions/unity-builder) action? I'm on Unity 2019.2.17f1, and 2019.3 will be dropping any day, so it would be nice not to be limited to 2019.2.11f1.

Also, out of curiosity, why is the `Unity - Builder` action configured with `with` while this action and `Unity - Activate` are both configured with `env`?

**Comment by webbertakken** [comment]
Hi @Rabadash8820, 

Thank you. Yes, this is planned. The implementation will probably be based on builder. I was going to wait for builder to become stable before starting it, but do feel free to send a PR if you feel up for it. 

The reason for working with the `with` keyword is that it's simpler for end users and that there's a JavaScript api in actions to convert `INPUT_*` to variables. Manually providing `INPUT_*` felt awkward to me and sticking with ENV too.

**Comment by Rabadash8820** [comment]
@webbertakken Thanks for the reply! Sounds good. I'm still quite new to GitHub Actions (the repo to which I just added your actions is actually my first use of them), but I'm getting more familiar with it, and would certainly rather than help on an already established project than build my own actions from scratch 😛

Regarding `with` vs `env`, does that mean you will probably migrate your other actions besides Builder to `with` also?

**Comment by webbertakken** [comment]
@Rabadash8820 sounds great!

> Regarding with vs env, does that mean you will probably migrate your other actions besides Builder to with also?

That would be goal if it works well (which it seems to do for now). Multi platform and multi version actions are highest priority for now. The `with` part we can do as part of moving to multi version/platform or after. I was thinking of not doing it before, in order to keep the older versions consistent.

**Comment by Rabadash8820** [comment]
Ok, thanks for the info. I'll leave it up to you whether you want to close this Issue now or wait til different version support is added.

**Comment by webbertakken** [comment]
@Rabadash8820 this is fixed in https://github.com/webbertakken/unity-test-runner/pull/10

Please let me know if you can work with this.

**Comment by Rabadash8820** [comment]
Hey @webbertakken, sorry for the radio silence! It looks like #10 is indeed the change that I requested, however its still not working for me because I am trying to build for Android, and your Actions [don't yet support Android](https://github.com/webbertakken/unity-actions/issues/12). That's beyond the scope of this Issue though.

I hope to do more troubleshooting of the Android build failures soon; I just haven't been super motivated to recently because Unity Cloud Build has been meeting my project's CI needs (I only started exploring GitHub Actions when Cloud Build didn't have the Unity Version that I needed, and now it does).
